test/integration/test_api.py

```diff
@@ -548,6 +548,45 @@ def test_get_ags_export(client, tmp_path):
             assert f'Project : {bgs_proj_id}' in metadata_text
 
 
+@pytest.mark.parametrize('bgs_loca_ids', [
+    ['20200205093728297908', '20200205093728297910'],  # One project
+    ['20200205093727287903', '20200205093728297906'],  # Two projects
+])
+@pytest.mark.xfail(IN_GITHUB_ACTIONS, reason="Upstream URL not available from Github Actions")
+def test_get_ags_export_multiple_ids(client, bgs_loca_ids):
+    """
+    Confirm that the endpoint can return the expected .zip.
+    """
+    # Arrange
+    # Define the borehole and project IDs and zipped AGS file to use for the test
+    bgs_proj_ids = {id_[:16] for id_ in bgs_loca_ids}  # unique ids when truncated to 16 digits
+    ags_file_names = {f'{id_}.ags' for id_ in bgs_proj_ids}
+    ags_metadata_file_name = 'FILE/BGSFileSet01/BGS_download_metadata.txt'
+
+    query = f'/ags_export/?bgs_loca_id={";".join(bgs_loca_ids)}'
+
+    # Act
+    with client as ac:
+        response = ac.get(query)
+
+    # Assert
+    assert response.status_code == 200
+    assert response.headers["Content-Disposition"] == 'attachment; filename="boreholes.zip"'
+    assert response.headers["Content-Type"] == "application/x-zip-compressed"
+    assert len(response.content) > 0
+
+    assert zipfile.is_zipfile(BytesIO(response.content))
```

Comment:
Again, here I am just testing that the correct files are in the zip and that the IDs appear in the text file.